helm/values.yaml

```diff
@@ -1,7 +1,7 @@
 # configure Connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v2.2.1
+  image: securesystemsengineering/connaisseur:v2.3.0
```

Comment:
changing the webserver does not qualify as minor change, does it? it is not a feature, does not extend or remove functionality. Why would you go for `v2.3.0`?

Reply:
I think it was a gut decision at the time of "being bigger" but you are correct. I shall revert this